sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -112,6 +112,11 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
       checkFilterPredicate('_1 === true, classOf[Eq[_]], true)
       checkFilterPredicate('_1 <=> true, classOf[Eq[_]], true)
       checkFilterPredicate('_1 !== true, classOf[NotEq[_]], false)
+
+      checkFilterPredicate(('_1 in(true)).asInstanceOf[Predicate],
```

Comment:
Another nit, usually we prefer the following styles if we have to wrap method arguments:

```scala
checkFilterPredicate(
  '_1.in(true).asInstanceOf[Predicate],
  classOf[UserDefinedByInstance[_, _]],
  true)
```

or

```scala
checkFilterPredicate(
  '_1.in(true).asInstanceOf[Predicate], classOf[UserDefinedByInstance[_, _]], true)
```